src/CoreWCF.ConfigurationManager/src/CoreWCF/Configuration/LocalClientSecuritySettingsElement.cs

```diff
@@ -0,0 +1,158 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.ComponentModel;
+using System.Configuration;
+
+namespace CoreWCF.Configuration
+{
+    internal sealed class LocalClientSecuritySettingsElement : ServiceModelConfigurationElement
+    {
```

Comment:
This class isn't needed. It's for client side config only. You can remove the property which references this.